<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenPositionField</class>
 <widget class="QWidget" name="OpenPositionField">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>40</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="label_logo">
     <property name="text">
      <string>logo</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_ticker">
     <property name="text">
      <string>ticker</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_value">
     <property name="text">
      <string>value</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_forToday">
     <property name="text">
      <string>forToday</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_forYesterday">
     <property name="text">
      <string>forYesterday</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_forAnEntierPeriod">
     <property name="text">
      <string>forAnEntierPeriod</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
